cmd/operator-opamp-bridge/metrics/reporter.go

```diff
@@ -122,16 +121,17 @@ func NewMetricReporter(
 
 	// Create some metrics that will be reported according to OpenTelemetry semantic
 	// conventions for process metrics (conventions are TBD for now).
-	reporter.processCpuTime, err = reporter.meter.Float64ObservableGauge(
+	reporter.processCpuTime, err = reporter.meter.Float64ObservableCounter(
```

Comment:
Is changing this metric type a breaking change?

Reply:
see below